feeds/wlan-ap/apc/src/src/ubus.c

```diff
@@ -14,10 +14,16 @@
 struct ubus_context *ubus_ctx = NULL;
 static struct blob_buf b;
 static struct blob_buf nb;
-static const char *ubus_path;
 timer *notify_timer;
 extern struct apc_iface * apc_ifa;
+/* Mandatorily Notify APC_State period */
 #define APC_NOTIFY_INTERVAL 10
```

Comment:
Unused define statement

Reply:
I use it in line 119:  cur_state == old_state && ucount < APC_NOTIFY_INTERVAL) {